src/smart-components/role/add-role-new/review.scss

```
@@ -0,0 +1,38 @@
+.sticky {
+	display: grid;
+    grid-template-rows: 1fr 250px; // second thing here is the height of the content area
+    
+	.title {
+		border-bottom: 1px solid var(--pf-global--BorderColor--300);
+		span { font-weight: 700; }
+	}
+	.title, .row {
+		display: grid;
+		grid-template-columns: 250px 250px 150px; // column width for each one
+		span { padding: 8px; }
+	}
+	.data {
+        overflow: auto;
+        -ms-overflow-style: none;  /* IE and Edge */
+        scrollbar-width: none;  /* Firefox */
+    }
+
+    /* Hide scrollbar for Chrome, Safari and Opera */
+    .data::-webkit-scrollbar {
+        display: none;
+    }
+	& + .sticky { margin-top: 18px; }
```

I think that this could be `--pf-global--spacer--md`
```suggestion
	& + .sticky { margin-top: var(--pf-global--spacer--md); }
```